### ISSUE TITLE
CSS rules in @media queries are being parsed unnecessarily

### ISSUE THREAD

**Issue by ianthedev** [keithclark/selectivizr#57]
It is understandable that when selectivizr was being introduced, the mobile web was not popular. While nowadays, websites use media queries for their mobile versions becomes a trend.

It's unproductive to scan and parse those thousands of lines of CSS codes within media queries for IE8 and below as they don't support media queries at all.

So in the next release please consider removing codes of media queries before parsing. Thanks.

**Comment by ianthedev** [comment]
Or maybe provide an option to enable media queries parsing because users may want to use Respond.js or something for IE8 and below.

**Comment by mike-engel** [comment]
Yes, please leave an alternate path, as my team is using Respond.js, and need those media queries.
